### PR TITLE
Ban every IP with bad request

### PR DESCRIPTION
**Pull request by alexbagirov**
refs #8

### CODE REVIEW

**Comment by alexbagirov** [comment]
Ready for review :tada:

File: request_handler.py [hunk 1]
```diff
@@ -149,6 +162,8 @@ def process(self):
         except OverflowError:
             log.error('server response was too large to encrypt')
             log.debug('response: {}'.format(response))
+            self.pr._ban_ip(address)
```
Comment (illright) [inline request_handler.py]: Don't ban here

File: processors.py [hunk 2]
```diff
@@ -193,6 +193,15 @@ def _close_session(self, ip):
 
         c.close()
 
+    def _ban_ip(self, ip):
+        """Заносит IP-адрес ip в таблицу tables с текущей датой"""
+        c = self.db.cursor()
+        stamp = datetime.now().timestamp()
+        c.execute('''INSERT INTO banned VALUES(%s, %s)''',
+                  (ip, str(stamp)))
```
Comment (illright) [inline processors.py]: Insert it as an integer

File: installer.py [hunk 3]
```diff
@@ -44,6 +44,9 @@ def create_database(self):
         c.execute('''CREATE TABLE key (pub_key text ARRAY,
                                        priv_key text ARRAY)''')
 
+        c.execute('''CREATE TABLE banned (ip text UNIQUE,
+                                          ban_time text)''')
```
Comment (illright) [inline installer.py]: Store it with the type `bigint`

**Review by illright — CHANGES_REQUESTED** [review]

**Review by illright — APPROVED** [review]

**Pull request merged**